src/middlewared/middlewared/migration/0009_nvram_attr_vms.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+
+from middlewared.plugins.vm.utils import FINAL_NVRAM_FOLDER_PATH, get_vm_nvram_file_name
+
+
+DEFAULT_NVRAM_FOLDER_PATH = '/var/lib/libvirt/qemu/nvram'
+LIBVIRT_QEMU_UID = 64055
+LIBVIRT_QEMU_GID = 64055
```

Comment:
Is there any way we can shift libvirt to have lower IDs (below 1000)? Otherwise this runs the risk of colliding with LDAP users (with potential for an LDAP user to be able to unexpectedly edit things as the libvirt-qemu).

Reply:
I noticed the libvirt-qemu user also has its homedir set to `/var/lib/libvirt`, does this mean we're still facing potential for other settings not getting properly migrated between versions?

Reply:
NOTE: I am not familiar with libvirt internals, but this seems like something that we are probably able to control.

Reply:
@anodos325 we do not want to preserve anything else in `/var/lib/libvirt` folder, it is just used by libvirt for running it's operations in our context. It has usecases but we are not using any of those so we should be safe.

I will work on changing the UID for libvirt user in a separate PR ✅